caracal/schema/transform_schema.yml

```diff
@@ -23,17 +23,17 @@ mapping:
         desc: Label of the output dataset.
         type: str
         required: false
-        example: 'corr' 
+        example: 'corr'
       rewind_flags:
-        desc: Rewind flags to specified version. Only works with a single-valued label_in (i.e., it does not loop over multiple comma-separated values in label_in).
+        desc: Rewind flags to specified version.
```

Comment:
@francescaLoi @o-smirnov I've just checked and in this branch it is still the case that flag rewinding in the transform worker only works with a single-valued `label_in` (I remind you that multiple `label_in` values can be given when concatenating input MS files along the frequency axis). So that warning should still be included in the description of `transform: rewind_flags`.

Reply:
should be ok now